### ISSUE TITLE
What's the meaning of inject_int and thres_int?

### ISSUE THREAD

**Issue by Ksky001** [LynnHo/AttGAN-Tensorflow#29]
why use extra attr_a_ and attr_b_ for attr_a and attr_b?

**Comment by LynnHo** [comment]
@Ksky001 `a_` is {-1,1} value to be sent into the generator, `a` is {0, 1} value to be used for calculating the sigmoid cross-entropy. You can ignore `thres_int` and just set it as 0.5 or 1, it's not an important parameter but hard to explain.
